app/views/onboarding/Onboarding4.js

```diff
@@ -97,4 +97,7 @@ const styles = StyleSheet.create({
   },
 });
 
-export default Onboarding;
+const mapDispatchToProps = dispatch => ({
```

Comment:
what do you think about using useDisptach and and useReducer hooks? I have been seeing most projects moving away from the connected component style now that hooks are available as they easier to reason about.

a blog post that explains the reasoning a bit if youre interested: https://thoughtbot.com/blog/using-redux-with-react-hooks

Reply:
I'm a fan of the hooks -- thought that onboaridng4 was a Class component still, I'll swap that over!